### ISSUE TITLE
Use navcodec for video processing

### ISSUE THREAD

**Issue by thom4parisot** [hughsk/stopmotion#1]
> https://www.npmjs.com/package/navcodec

Benefits:
- it would avoid using child_process.spawn;
- it would compile on every system;
- it avoids relying on an implicit system dependency;
- easier to deal with

Cons:
- none?

**Comment by hughsk** [comment]
@oncletom in that case you should submit a pull request – if you can do it without breaking the current API I'm all for it.

**Comment by thom4parisot** [comment]
Agreed — it was a suggestion. I kind of supposed you would be faster at implementing such a thing if it was in your roadmap.

I will try to have a go at it anyway :-)

**Comment by benwiley4000** [comment]
Seems navcodec relies on ffmpeg being installed at the system level
